src/unix/notbsd/linux/other/b64/x86_64.rs

```diff
@@ -164,7 +167,11 @@ s! {
         pub start_stack: ::c_ulonglong,
         pub signal: ::c_longlong,
         __reserved: ::c_int,
+        #[cfg(target_pointer_width = "32")]
+        __pad1: u32,
         pub u_ar0: *mut user_regs_struct,
+        #[cfg(target_pointer_width = "32")]
+        __pad2: u32,
```

Comment:
Considering the [definition](https://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/unix/sysv/linux/x86/sys/user.h;hb=a4cc02c7413af4826fe8d13b6250c3fc47f15699#l85) of the user struct,  it's seems odd to add a pad before the field, but without it, the test fails...

Reply:
*shrug*